src/main/java/Todo.java

```diff
@@ -0,0 +1,11 @@
+public class Todo extends Task{
+    public Todo(String description){
```

Comment:
Standardize whether after the function/class name, will you leave a space before typing the opening bracket or not